tests/cuda_test/test_kernel_semantic.py

```diff
@@ -121,4 +121,103 @@ def kernel_call():
     # Check that the error is correct
     error_info = [*errors.error_info_map.values()][0][0]
     assert error_info.symbol.func  == 'kernel_call'
-    assert UNVALID_KERNEL_CALL_TP_BLOCK == error_info.message
+    assert INVALID_KERNEL_CALL_TP_BLOCK == error_info.message
+
+@pytest.mark.parametrize( 'language', [
+        pytest.param("ccuda", marks = pytest.mark.ccuda)
+    ]
+)
+def test_invalid_kernel_config(language):
+    def invalid_kernel_config():
+        @kernel
+        def kernel_call():
+            pass
+        kernel_call[1.0, 2.0]()
```

Comment:
Why are these indices floats?

Reply:
The indices should be integers. I intentionally provided a float values as an invalid types to test that the appropriate error is raised.

Reply:
:facepalm: of course. Sorry